cmd/antrea-agent/agent.go

```diff
@@ -753,10 +789,15 @@ func run(o *Options) error {
 	}
 
 	if enableMulticlusterGW {
-		mcInformerFactory.Start(stopCh)
-		go mcRouteController.Run(stopCh)
+		mcInformerFactoryWithOption.Start(stopCh)
+		go mcDefaultRouteController.Run(stopCh)
+		if notEncapMode {
```

Comment:
Could we check` mcPodRouteController != nil` instead?

And if so we can remove `notEncapMode`.

Reply:
Done